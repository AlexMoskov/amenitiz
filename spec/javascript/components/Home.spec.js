import React from 'react'
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from 'components/Home'

configure({ adapter: new Adapter() });

describe('Home', () => {
    const props = {}
    const component = shallow(<Home {...props} />)

    it('renders Home component', () => {
        expect(toJson(component)).toMatchSnapshot()
    })
})
