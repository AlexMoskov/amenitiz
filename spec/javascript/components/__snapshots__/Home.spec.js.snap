// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Home component 1`] = `
<div
  className="vw-100 vh-100 primary-color d-flex align-items-center justify-content-center"
>
  <div
    className="jumbotron jumbotron-fluid bg-transparent"
  >
    <div
      className="container secondary-color"
    >
      <h1
        className="display-4"
      >
        Best Products
      </h1>
      <p
        className="lead"
      >
        A curated list of products for the best shopping.
      </p>
      <hr
        className="my-4"
      />
      <Link
        className="btn btn-lg custom-button"
        role="button"
        to="/products"
      >
        View Products
      </Link>
      <Link
        className="btn btn-lg btn-primary right"
        to="/purchases"
      >
        Your purchases
      </Link>
    </div>
  </div>
</div>
`;
